apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -275,6 +275,11 @@ export const EventSetupTab = (
                               teamName: e.teamName,
                             }),
                           });
+                          const newDefaultValue = eventLocationType.defaultValueVariable ? "" : undefined;
```

Comment:
As per my understanding all integrations have some sort of defaultValueVariable, I don't think this is the right way to fix this.